cli/add_a_pool.md

```diff
@@ -0,0 +1,20 @@
+# Adding a pool
+Following the other examples in this chapter, we would add a pool using
+the following command:
+```
+  $ nipap pool add name link-networks \
+     description "Link networks for my test network" \
+     default-type  assignment \
+     ipv4_default_prefix_length 30 \
+     ipv6_default_prefix_length 112
+```
```

Comment:
Try to include example output so users feel at home and it's consistent with the rest of the docs